packages/model/addon/-private/system/many-array.js

```diff
@@ -166,6 +166,13 @@ export default EmberObject.extend(MutableArray, DeprecatedEvented, {
     return false;
   },
 
+  toArray() {
+    // By using `get()`, the tracking system knows to pay attention to changes that occur.
+    get(this, '[]');
```

Comment:
@pzuraq 👋 I'm noticing that caching on `this` once when accessed doesn't pass the tests I added here. We have to consume every time it is accessed. 

I'm thinking this might be expected since `sortBy` (through `toArray`) returns a new array each time.  What do you think?

Reply:
Discussed this in the data meeting but wanted to note it down here, the issue is that `length` also needs to be instrumented here, along with `objectAt`. This because `objectAt` is not used when the length is 0, but length is read to check that.